RELEASE.md

```diff
@@ -197,8 +197,8 @@ To make a macOS release, go to macOS build machine:
 - Build machine must have:
   - Apple-trusted `Developer ID Application: Freedom of the Press Foundation (94ZZGGGJ3W)` code-signing certificates installed
 - Apple account must have:
-  - A valid application password for `altool` in the Keychain. You can verify
-    this by running: `xcrun altool --notarization-history 0 -u "<email>" -p "@keychain:altool"`
+  - A valid application password for `notarytool` in the Keychain. You can verify
```

Comment:
Would it make sense to say that, if the application password is not stored there, that the user can store it with `xcrun notarytool store-credentials dz-notarytool-release-key --apple-id <email> --team-id <team ID>`?

Reply:
Sure. I just added something like that in 20a05d39.

Reply:
Nice, thanks.